src/debugpy/common/messaging.py

```diff
@@ -451,7 +472,7 @@ def __init__(self, channel, seq, json=None):
         """
 
     def __str__(self):
-        return json.repr(self.json) if self.json is not None else repr(self)
+        return json.repr(self.json).__str__() if self.json is not None else repr(self)
```

Comment:
Shouldn't you call `str(json.repr(self.json))` rather than using `__str__`? https://docs.python.org/3.11/reference/datamodel.html#object.__str__

Reply:
Oh that looks better. Yeah, I'll do that